### PR TITLE
Add lint step to CI.

### PR DESCRIPTION
**Pull request by fitzy101**

### CODE REVIEW

File: lib/pups/exec_command.rb [hunk 1]
```diff
@@ -1,124 +1,128 @@
-require 'timeout'
+# frozen_string_literal: true
 
-class Pups::ExecCommand < Pups::Command
-  attr_reader :commands, :cd
-  attr_accessor :background, :raise_on_fail, :stdin, :stop_signal
+require 'timeout'
+require 'English'
```
Comment (SamSaffron) [inline lib/pups/exec_command.rb:4]: what is this thing?

**Comment by SamSaffron** [comment]
Except for the `English` thing, stuff looks great

File: lib/pups/exec_command.rb (hunk 1 above)
Comment (fitzy101) [reply lib/pups/exec_command.rb:4]: this is due to the change from the `$?` global to `$CHILD_STATUS`, see: https://github.com/rubocop/rubocop/issues/1747 and https://ruby-doc.org/stdlib-2.7.1/libdoc/English/rdoc/English.html

alternatively I could ignore that rubocop change?

Comment (SamSaffron) [reply lib/pups/exec_command.rb:4]: I see, TIL. Sure this change is fine.

**Review by SamSaffron — APPROVED** [review]

**Pull request merged**